### ISSUE TITLE
1.10 Release?

### ISSUE THREAD

**Issue by LukvonStrom** [Poweruser/Migot#2]
Is there a 1.10 Release planned?

**Comment by Poweruser** [comment]
No. Maybe at some point, when I am directly involved into running a server again. And it will be only for that specific version in use, because I am tired of having to reinvent the improvements for almost every version.  I barely play the game anymore these days. And when I put so much time into this, it has to be personally worth it. I won't do it, when someone else asks me to, I (did) do it, because I want it myself. And as nobody else is doing it, there is no other option than to do it myself. I am just sharing what I do. I do not intent to serve a public interest.

**Comment by maximvarentsov** [comment]
Looks you patches, thinking about try port it on 1.11 :)
